qa/suites/fs/upgrade/mds_upgrade_sequence/fail_fs/yes.yaml

```diff
@@ -0,0 +1,8 @@
+teuthology:
+  variables:
+    fail_fs: true
+overrides:
```

Comment:
drop these overrides since we're setting the config later.

Reply:
Done